lib/tasks/temporary/hacktoberfest_badges.rake

```diff
@@ -0,0 +1,34 @@
+# TODO: Will be removed my @mstruve once badges are rewarded for 2020
+# rubocop:disable  Metrics/BlockLength
+task award_hacktoberfest_badges: :environment do
+  badge_id = Badge.find_by(slug: "hacktoberfest-2020")&.id
+  user_ids = UserSubscription.where(
+    user_subscription_sourceable_id: 529_344,
+    created_at: 2.hours.ago..Time.current,
+  ).pluck(:subscriber_id)
+
+  User.where(id: user_ids).includes(:identities).find_each do |user|
+    next if user.badges.exists?(id: badge_id)
+
+    identity = user.identities.github.first
+    next unless identity
+
+    github_user_id = identity.uid
+    response = HTTParty.get(
+      "https://hacktoberfest.digitalocean.com/api/state/#{github_user_id}",
+      headers: { "Authorization" => ENV["HACKTOBERFEST_AUTH_KEY"] },
+    )
+    if response.parsed_response == "completed"
+      BadgeAchievement.create(
+        user_id: user.id,
+        badge_id: badge_id,
+        rewarding_context_message_markdown: "Congrats!",
```

Comment:
The only question I have about the markdown is whether we need to say something about getting this badge from hacktoberfest? Just thinking about the email I got that told me I got the badge and I wonder if we should be trying to maintain "copy clarity"?

Reply:
This is the default message but the email and how it appears in your profile comes out with Hacktoberfest. 
Email:
![Screen Shot 2020-12-03 at 3 10 34 PM](https://user-images.githubusercontent.com/1813380/101082637-6232b900-3571-11eb-9d63-5e908000c812.png)
Notifications:
![Screen Shot 2020-12-03 at 3 11 15 PM](https://user-images.githubusercontent.com/1813380/101082674-6eb71180-3571-11eb-97d5-c7d427c40f84.png)

Reply:
Ah, gotcha 👍